draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -536,6 +540,29 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
          (see first_unvalidated_packet_acknowledged in <xref target="sec-QLOG"></xref>).</t>
         </list></t> <!-- End of list of actions -->
 
+        <t>When the flow is controlled using BBR, the Careful Resume is implemented
+           by setting the pacing rate from the saved congestion control parameters,
+           with the following precautions:</t>
+           <t><list style="symbols">
+              <t>The flags "carefully-resuming" and "carefully-evaluating" are
+                 added to the BBR state, and initialized
+                 to "False" when the BBR flow starts.</t>
+              <t>Careful Resume is only activated if a BBR flow is in the Startup state</t>
+              <t>The sender starts the "evaluation" phase at the beginning of a round,
+                 and sets the "carefully-resuming" and "carefully-evaluating" flags to "True".
+              <t>When the "carefully-resuming" is set, the sender sets the pacing rate
+                 to the larger of the nominal pacing rate (BBR.bw times BBRStartupPacingGain)
+                 and the remembered value of BBR.bw_max. If the nominal pacing rate
+                 is higher, the "carefully-resuming" flag is reset to False.
+              <t>The "carefully-resuming" flag is in any case reset to False at the
+                 beginning of the next round.
+              <t>The "carefully-evaluating" flag is reset to False two rounds after
+                 it is set, i.e., after all the packets sent when "carefully resuming"
+                 have been received and acknowledged by the peer. At that stage,
+                 the measured delivery rate is expected to reflect the capacity of
+                 the path.
+              </t>
+           </list></t>
```

Comment:
This new text incorporates the results from the discussion. Notable points:

* On entering the Evaluation phase, we only change the "pacing rate", which is a transient variable. This means that state variables like BBR.bw or BBR.bw_max will only be updated by actual measurements of delivery rates. That is safer.
* The pacing rate is set to the remembered value. The previous version was setting the state variable "BBR.bw" to half that value, which resulted in a pacing rate of BBR.bw times BBRStartupPacingGain, i.e., 1/2 of the remembered value time 2.77. The new version is more conservative.
* We do not mention entering Drain (or Safe Retreat) here. This is by design. BBR states that Drain entered upon packet loss, or if the delivery rate does not increase for 3 rounds. This is actually already problematic; I am proposing also entering Drain upon RTT increase in a different draft.